src/servers/middlewares/auth-router.ts

```diff
@@ -58,7 +58,11 @@ authRouter.post(
         cookies = setCookie.parse(loginResp.cookiesToSendBackToClient)
         kratosUserId = loginResp.kratosUserId
       } catch (err) {
-        recordExceptionInCurrentSpan({ error: err })
+        if (err instanceof Error) {
+          recordExceptionInCurrentSpan({ error: err })
+        } else {
+          recordExceptionInCurrentSpan({ error: { message: "Unknown error" } as Error })
```

Comment:
this casting sounds awkward to me

Reply:
any suggestions? i was just trying to be extra cautious because cookies is type `any` and I wanted to record the case that it was not an instanceof Error